### PR TITLE
Add section about Clixon and Clixon Controller

### PR DESCRIPTION
**Pull request by krihal**
Added section about the Clixon and Clixon Controller open source tools.

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
I am not fully following what clixon does from the docs, can you provide a little more info?

**Comment by krihal** [comment]
Updated the description and tried to make it more clear.

File: README.md [hunk 1]
```diff
@@ -250,6 +251,14 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 
  - [Using Chef Client with Cisco NX-OS](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/sw/7-x/programmability/guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x_chapter_01110.html) - Using Chef Client with Cisco NX-OS
 
+## Clixon
+
+ - [Home Page](https://clicon.org) - Clicon home page with links to the different projects with code and documentation.
+
+   - [Clixon](https://github.com/clicon/clixon) - Clixon provides an interactive CLI, NETCONF, and RESTCONF configuration interface for a YANG-based device. It generates the interfaces from YANG models, uses an embedded database with transaction support. That means you can feed Clixon with one or more YANG specifications and it will create an interactive CLI together with RESTCONF and NETCONF interfaces and an configuration database which can be added to any networking device.
+
+   - [Clixon Controller](https://github.com/clicon/clixon-controller) - The Clixon controller is an open-source tool for network automation of devices based on NETCONF and YANG.
```
Comment (itdependsnetworks) [inline README.md:260]:
```suggestion
 - [Home Page](https://clicon.org) - Clixon home page with links to the different projects with code and documentation.
   - [Clixon](https://github.com/clicon/clixon) - Clixon provides an interactive CLI, NETCONF, and RESTCONF configuration interface for a YANG-based device. 
   - [Clixon Controller](https://github.com/clicon/clixon-controller) - The Clixon controller is an open-source tool for network automation of devices based on NETCONF and YANG.
```

Comment (itdependsnetworks) [reply README.md:260]: Does this work?

Comment (krihal) [reply README.md:260]: Sure, that'll be fine.

**Pull request merged**